packages/mobile/package.json

```diff
@@ -105,7 +105,7 @@
     "react-native-flag-secure-android": "git://github.com/kristiansorens/react-native-flag-secure-android#e234251",
     "react-native-fs": "^2.16.6",
     "react-native-gesture-handler": "^1.6.1",
-    "react-native-geth": "https://github.com/celo-org/react-native-geth#1ba8817",
+    "react-native-geth": "https://github.com/celo-org/react-native-geth#0fcf62f",
```

Comment:
If you're using this commit be sure to merge your RN-geth PR without squash so it still refers to master. We've made this mistake in the past :)

Reply:
Yep! I've merged the PR and updated the sha.